x-pack/plugins/security_solution/public/common/components/sourcerer/get_sourcerer_data_view.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { DataView } from '@kbn/data-views-plugin/public';
+import { useSourcererDataView } from '../../containers/sourcerer';
+import { useKibana } from '../../lib/kibana';
+import { SourcererScopeName } from '../../store/sourcerer/model';
+
+// TODO: Define what the difference is between this and the sourcererDataView returned
```

Comment:
@michaelolo24, do you remember this by any chance?

Reply:
Yea. I think the real difference is on line 28. And I should have specified. It's differentiating this return from `use_sourcerer_data_view` here: 

x-pack/plugins/threat_intelligence/public/modules/indicators/hooks/use_sourcerer_data_view.ts

I just see us recreating the `DataView` instance a bunch of places, so thought we could replace it with this. I just need to investigate how that differs from `sourcererDataView`

Reply:
Understood. This hook converst `DataviewSpec` to `Dataview`.